pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -166,7 +166,7 @@ func (ks *scaler) handleScaleToZero(ctx context.Context, pa *pav1alpha1.PodAutos
 		}
 		ks.enqueueCB(pa, activationTimeout)
 		return scaleUnknown, false
-	} else if pa.Status.IsReady() { // Active=True
+	} else if pa.Status.IsActive() { // Active=True
```

Comment:
Why this change?

Reply:
It's a result of propagating the metric status to PA, so the `IsReady()` checks for metric ready condition too. However, since we are reducing the scope of this PR to updating metric status, this change isn't needed for this PR.